### PR TITLE
Resolve the merge conflicts during the ecdsa merge.

### PR DESCRIPTION
**Pull request by dlorenc**

### CODE REVIEW

File: cmd/cli/verify.go [hunk 1]
```diff
@@ -63,6 +63,9 @@ func Verify() *ffcli.Command {
 			if !*checkClaims {
 				fmt.Fprintln(os.Stderr, "Warning: the following claims have not been verified:")
 			}
+			if os.Getenv(tlog.Env) != "1" {
```
Comment (priyawadhwa) [inline cmd/cli/verify.go:66]: I think this should be `== 1`?

File: cmd/cli/sign.go [hunk 2]
```diff
@@ -1,5 +1,5 @@
 /*
-Copyright The Rekor Authors
+Copyright The Sigstore Authors
```
Comment (priyawadhwa) [inline cmd/cli/sign.go:2]: oops 😅

File: pkg/cosign/tlog/verify.go [hunk 3]
```diff
@@ -32,82 +30,92 @@ import (
...
+	signature, err := base64.StdEncoding.DecodeString(sp.Base64Signature)
+	if err != nil {
+		return errors.Wrap(err, "decoding base64 signature")
+	}
+	re := rekorEntry(sp.Payload, signature, pubKey)
+	entry := &models.Rekord{
+		APIVersion: swag.String(re.APIVersion()),
+		Spec:       re.RekordObj,
+	}
+	// j, _ := entry.MarshalJSON()
```
Comment (priyawadhwa) [inline pkg/cosign/tlog/verify.go:80]: nit: leftover debugging

**Review by priyawadhwa — APPROVED** [review]
LGTM thank you for fixing !

File: pkg/cosign/tlog/verify.go (hunk 3 above)
Comment (dlorenc) [reply pkg/cosign/tlog/verify.go]: Nice!

File: cmd/cli/verify.go (hunk 1 above)
Comment (dlorenc) [reply cmd/cli/verify.go]: Whoops

**Pull request merged**